Backend/router/v1/user.js

```diff
@@ -62,14 +69,18 @@ router.get('/signin', (req, res) => {
     {
       res.json({msg: "User does not exist"});
     } else {
-      let dbPassword = result[0].dataValues.password;
-      let hashPassword = crypto.createHash("sha512").update(req.query.password).digest("hex");
-      if(dbPassword == hashPassword) {
-        req.session.email = req.query.email;
-        res.json({msg: req.session.email});
+      if (req.body.password.length < MIN_PASSWORD_LENGTH || req.body.password.length > MAX_PASSWORD_LENGTH ) {
```

Comment:
I see this if statement repeated here and above. Can you refactor this out?

Reply:
I will make a password_length_check function.

Reply:
No need to update the comment on what you 'will' be doing. Just resolve this once you're actually done :)

Reply:
Oh, yes!